### PR TITLE
Misc. typo fixes

### PR DESCRIPTION
**Pull request by gunnarmorling**
A few fixes and one link.

### CODE REVIEW

File: documented-extensions.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Support for any extension is OPTIONAL. When an extension definition uses
 MAY), this usage only applies to events that use the extension.
 
 Extensions always follow a common placement strategy for in-memory formats (e.g.
-[JSON](json-format.md), XML, Protobuffer) that are decided by those
+[JSON](json-format.md), XML, [Protocol Buffers(protobuf-format.md)) that are decided by those
```
Comment (duglin) [inline documented-extensions.md]: Missing a `]`

Comment (duglin) [reply documented-extensions.md]: @n3wscott are you ok with this change to "Protobuffer" ?

Comment (gunnarmorling) [reply documented-extensions.md]: > Missing a ]

Oops, so much for fixing typos ;) Updated it.

>  "Protobuffer"

"Protocol Buffers" is the official name as per the [project page](https://developers.google.com/protocol-buffers/). I've seen "protobuf" [for short](https://github.com/protocolbuffers/protobuf), but not really "Protobuffer".

**Comment by gunnarmorling** [comment]
Updated the commit as per @duglin's comment and also fixed one more typo in another file.

File: documented-extensions.md (hunk 1 above)
Comment (duglin) [reply documented-extensions.md]: Yep - I just wanted a googler's +1 :-)

**Comment by duglin** [comment]
LGTM

Since these are type typos, once @n3wscott agrees with the protobuf change I'll merge it and not wait for the next call - then these can get into the v0.3 release.

File: documented-extensions.md (hunk 1 above)
Comment (n3wscott) [reply documented-extensions.md]: protobuf or Protocol Buffers. also needs a `]`

**Comment by duglin** [comment]
Got an off-line LGTM from @n3wscott so since this is just typos I'm going to merge it.
thanks @gunnarmorling

**Pull request merged**